frontend/src/app/types/notification/admin-notification.ts

```diff
@@ -0,0 +1,10 @@
+export interface AdminNotifcation {
```

Comment:
Small thing but other type defs have been moved out of app dir in ./src/types

Reply:
moved